docs/zh_cn/recommended_topics/deploy/mmdeploy_guide.md

```diff
@@ -4,7 +4,7 @@
 
 MMDeploy 是 [OpenMMLab](https://openmmlab.com/) 模型部署工具箱，**为各算法库提供统一的部署体验**。基于 MMDeploy，开发者可以轻松从训练 repo 生成指定硬件所需 SDK，省去大量适配时间。
 
-更多介绍和使用指南见 https://github.com/open-mmlab/mmdeploy/blob/dev-1.x/docs/zh_cn/get_started.md
+更多介绍和使用指南见 https://github.com/open-mmlab/mmdeploy/blob/main/docs/zh_cn/get_started.md
```

Comment:
may use the readthedocs link https://mmdeploy.readthedocs.io/zh_CN/latest/get_started.html